pandas/core/base.py

```diff
@@ -990,7 +990,7 @@ def factorize(self, sort=False, na_sentinel=-1):
         from pandas.core.algorithms import factorize
         return factorize(self, sort=sort, na_sentinel=na_sentinel)
 
-    def searchsorted(self, key, side='left'):
+    def searchsorted(self, key, side='left', sorter=None):
```

Comment:
I don't believe so.  If you look at the `numpy` call to `searchsorted`, it unfortunately passes them in as if they were positional arguments.